src/test/rustdoc/inline_local/trait-vis.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2012-2013 The Rust Project Developers. See the COPYRIGHT
```

Comment:
I think date can be changed.

Reply:
Oops, that's what i get for copying headers blindly. :sweat_smile: